pipeline/extract_to_s3.py

```diff
@@ -1,10 +1,10 @@
 import os
 import logging
 import timeit
-from pipeline.extract_generation import main as extract_generation
+from extract_generation import main as extract_generation
```

Comment:
here the `pipeline.` is not needed but for the other scripts it will be needed. Pytest messes with paths, and so when it tries to run scripts it cannot find sibling modules without specifying the name of the dir they are already in.

It looks silly, but it is needed in some instances. Speak to me if this doesn't make sense.